ios/src/platform_ios.mm

```diff
@@ -1,15 +1,21 @@
 #ifdef PLATFORM_IOS
 
 #import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
 #import <utility>
 #import <cstdio>
 #import <cstdarg>
 #import <fstream>
 #import <regex>
 
+#include <iostream>
```

Comment:
Yep, we can clean up those a bit and use `import` instead, even though those are in the same compilation unit so there should be no risk of multiple inclusion.